includes/wf_crm_webform_postprocess.inc

```diff
@@ -1513,6 +1513,25 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
         'entity_table' => 'civicrm_contribution',
       );
     }
+    // KG - start
+    // LineItems
+    $fid = "civicrm_1_lineitem_1_contribution_line_total";
+    if (isset($this->enabled[$fid])) {
+      foreach ($this->data['lineitem'][1]['contribution'] as $l => $lineitem) {
+        $fid = "civicrm_1_lineitem_{$l}_contribution_line_total";
```

Comment:
`$1`?

Reply:
Ah - that's actually a letter lower case 'l' (not a number 1); we're using $c for contact and $n in other places - I used $l for the lineitem number. I'll make that $n (easier to read).

Reply:
Ok - done